src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a find / query user command.
+ * Inherits from command class.
+ */
+public class FindCommand extends Command {
```

Comment:
`CommandFinder`?

_Actually ... the class name is vague as it sounds like a verb (even though I know it was meant as a noun)_

Reply:
Yea honestly agree that its vague as "command" is a noun as well